src/Providers/DataGrid/Wijmo/Columns/DropdownColumn.ts

```diff
@@ -160,66 +160,80 @@ namespace Providers.DataGrid.Wijmo.Column {
-                rowNumber,
-                this.provider.index
-            );
-            const currentSel = this.grid.provider.selection;
 
-            // check if the first selected parent cell has an undo action on undo stack
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            const existingUndoAction: any =
-                this.grid.features.undoStack.stack._stack.find(
-                    (data: GridEditAction) =>
-                        data.col === currentSel.leftCol &&
-                        data.row === currentSel.topRow
+            // if the child dropdown value is already empty, we don't want register the undo action and trigger cell value change event again
+            if (column && currentValue !== '') {
+                // get original value of the child cell
+                const originalValue = this.grid.features.dirtyMark.getOldValue(
+                    rowNumber,
+                    this.grid.getColumnByIndex(this.provider.index).config
+                        .binding
                 );
 
-            if (existingUndoAction) {
-                // check if current child cell has an undo action on undo stack
-                const existingEditActionForColRow =
-                    existingUndoAction?._actions?.find(
-                        (action: GridEditAction) =>
-                            action.col === this.provider.index &&
-                            action.row === rowNumber
-                    ) ?? false;
-
-                // only add child undo action if it doesn't already exist. we don't want duplicated actions
-                if (existingEditActionForColRow === false) {
-                    // add new child action into existing parent action in order
-                    existingUndoAction.addChildAction(
-                        new GridEditAction(
-                            this.grid,
-                            new wijmo.grid.CellRangeEventArgs(
-                                this.grid.provider.cells,
-                                cellRange
-                            )
-                        )
+                const cellRange = new wijmo.grid.CellRange(
+                    rowNumber,
+                    this.provider.index
+                );
+                const currentSel = this.grid.provider.selection;
+
+                // filter the parent cell has an undo actions on undo stack
+                const filteredUndoAction: wijmo.undo.UndoableAction[] =
+                    this.grid.features.undoStack.stack._stack.filter(
+                        (data: GridEditAction) =>
+                            data.col === currentSel.leftCol &&
+                            data.row === currentSel.topRow
                     );
+
+                // get the lastest undo action filtered if any
+                // eslint-disable-next-line @typescript-eslint/no-explicit-any
+                const existingUndoAction: any =
```

Comment:
@OS-giulianasilva any way to avoid the any? I think in the past Gabriel did something about it, not sure what. 

I believe he started to create types somewhere, but I might be wrong.

Reply:
@rugoncalves any insight?

Reply:
@BenOsodrac I could not find this file that you mentioned. For now, I will proceed with the merge, but next week I can evaluate it with Rúben and correct it if necessary.